steps/pvs_like_case_study/implementations/pvs_like_r/README.md

```diff
@@ -0,0 +1,49 @@
+# PVS-like case study
+
+This directory contains a case study emulating the Census Bureau's Person Identification
```

Comment:
Check w/ @zmbc if this is ok to say

Reply:
Fine for now; I wrote it. I want to change a bunch of things like this at some point in the near future but it will be easier if I do it all at once.

Reply:
It was already public in the vivarium_research_prl repo, nevermind that this README will be invalid with the next PR anyway!